Doc about InstallTracker files and `install.root` (#9948)
rust-lang/cargo: src/doc/src/reference/config.md

Sets the path to the root directory for installing executables for [`cargo
install`]. Executables go into a `bin` directory underneath the root.

To track information of installed executables, some extra files, such as

> `.crates.toml` and `.crates2.json`, would also be created under this root.

Reviewer: Thanks, might I suggest a small wording change?

```suggestion
`.crates.toml` and `.crates2.json`, are also created under this root.
```
